lsp-pyls.el

```diff
@@ -151,7 +151,7 @@ complexity."
   :group 'lsp-pyls
   :package-version '(lsp-mode . "6.1"))
 
-(defcustom lsp-pyls-plugins-pylint-args t
```

Comment:
it should be [] if you want to pass an empty array. () is the same as nil.

Reply:
nil was okay as well, but this makes sense. Fixed